### PR TITLE
feat: Add QR code to assessment student link

### PR DESCRIPTION
**Pull request by jonatanschroeder**
Can be useful for in-class worksheets if instructor shows the code on the screen.

screenshot

### CODE REVIEW

**Comment by nwalters512** [comment]
hmm, I need to make that bot ignore PRs that don't touch the `packages/` directory...

**Comment by nwalters512** [comment]
Ok, I don't think there's a way to make the bot comment only for PRs that affect `packages`: https://github.com/changesets/bot/issues/11. I've uninstalled the bot for now.

File: pages/instructorAssessmentSettings/instructorAssessmentSettings.ejs [hunk 1]
```diff
@@ -97,9 +102,16 @@
...
                         <span class="input-group">
                           <span readonly class="form-control form-control-sm"><%= studentLink %></span>
-                          <button type="button" class="input-group-prepend btn btn-sm btn-outline-secondary btn-copy" data-clipboard-text="<%= studentLink %>">
+                          <button type="button" class="input-group-append btn btn-sm btn-outline-secondary btn-copy" data-clipboard-text="<%= studentLink %>">
                             <i class="far fa-clipboard"></i>
                           </button>
+                          <button type="button" title="Student Link QR Code"
+                                  class="input-group-append btn btn-sm btn-outline-secondary js-student-link-qrcode-button">
+                            <i class="fas fa-qrcode"></i>
+                          </a>
```
Comment (nwalters512) [inline pages/instructorAssessmentSettings/instructorAssessmentSettings.ejs:111]: Two changes:

- Uses the correct markup from https://getbootstrap.com/docs/4.6/components/input-group/#button-addons
- Fixes the closing tag

```suggestion
                          <div class="input-group-append">
                            <button type="button" class="btn btn-sm btn-outline-secondary btn-copy" data-clipboard-text="<%= studentLink %>">
                              <i class="far fa-clipboard"></i>
                            </button>
                            <button type="button" title="Student Link QR Code"
                                  class="btn btn-sm btn-outline-secondary js-student-link-qrcode-button">
                              <i class="fas fa-qrcode"></i>
                            </button>
                          </div>
```

Can you make the same changes to the other page too?

Comment (jonatanschroeder) [reply pages/instructorAssessmentSettings/instructorAssessmentSettings.ejs]: Done.

**Review by nwalters512 — APPROVED** [review]

**Pull request merged**